modules/building-memcached-operator-using-osdk.adoc

```diff
@@ -289,22 +334,37 @@ Memcached CR.
 .. Create the example `Memcached` CR that was generated at
 `deploy/crds/cache_v1alpha1_memcached_cr.yaml`:
 +
+[source,terminal]
 ----
 $ cat deploy/crds/cache_v1alpha1_memcached_cr.yaml
+----
++
+.Example output
+[source,terminal]
+----
 apiVersion: "cache.example.com/v1alpha1"
 kind: "Memcached"
 metadata:
   name: "example-memcached"
 spec:
   size: 3
-
+----
++
+[source,terminal]
+----
```

Comment:
In the following step, the `oc apply -f`command is a separate step, if you want to change this one while you are in there.

Reply:
OK, re-wrote the step a bit. Was a little weird because it was just showing a `cat` then an `oc create`.